lib/node_modules/@stdlib/array/base/slice/docs/types/index.d.ts

```diff
@@ -37,170 +37,16 @@ import { Collection, Complex128Array, Complex64Array } from '@stdlib/types/array
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Int8Array = require( '@stdlib/array/int8' );
-*
-* var x = new Int8Array( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 3 );
-* // returns <Int8Array>[ 1, 2, 3 ]
-*/
-declare function slice( x: Int8Array, start: number, end: number ): Int8Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Uint32Array = require( '@stdlib/array/uint32' );
-*
-* var x = new Uint32Array( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 3 );
-* // returns <Uint32Array>[ 1, 2, 3 ]
-*/
-declare function slice( x: Uint32Array, start: number, end: number ): Uint32Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Uint16Array = require( '@stdlib/array/uint16' );
-*
-* var x = new Uint16Array( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 3 );
-* // returns <Uint16Array>[ 1, 2, 3 ]
-*/
-declare function slice( x: Uint16Array, start: number, end: number ): Uint16Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Uint8Array = require( '@stdlib/array/uint8' );
-*
-* var x = new Uint8Array( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 3 );
-* // returns <Uint8Array>[ 1, 2, 3 ]
-*/
-declare function slice( x: Uint8Array, start: number, end: number ): Uint8Array;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
-*
-* @example
-* var Uint8ClampedArray = require( '@stdlib/array/uint8c' );
-*
-* var x = new Uint8ClampedArray( [ 1, 2, 3 ] );
-*
-* var out = slice( x, 0, 3 );
-* // returns <Uint8ClampedArray>[ 1, 2, 3 ]
-*/
-declare function slice( x: Uint8ClampedArray, start: number, end: number ): Uint8ClampedArray;
-
-/**
-* Returns a shallow copy of a portion of an array.
-*
-* @param x - input array
-* @param start - starting index (inclusive)
-* @param end - ending index (exclusive)
-* @returns output array
 *
 * @example
-* var Complex128Array = require( '@stdlib/array/complex128' );
+* var Complex64Array = require( '@stdlib/array/complex64' );
 *
-* var x = new Complex128Array( [ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ] );
+* var x = new Complex64Array( [ 1.0, 2.0, 3.0, 4.0 ] );
 *
 * var out = slice( x, 0, 3 );
-* // returns <Complex128Array>[ 1.0, 2.0, 3.0, 4.0, 5.0, 6.0 ]
+* // returns <Complex64Array>[ 1.0, 2.0, 3.0 ]
```

Comment:
@Planeshifter The changes to this example should be reverted, as they are not correct.

Reply:
Fixed.